bin/cli.js

```diff
@@ -258,7 +259,8 @@ function createOutputFromOptions(program, res) {
 		const circular = res.circular();
 
 		output.circular(spinner, res, circular, {
-			json: program.json
+			json: program.json,
+			index: program.index
```

Comment:
Can you rename the `index` in the object to `showIndex`, or better extract the object to `outputOption`? It would be misunderstood with other "index" concepts like the `idx` in `output.js`

Or maybe call it `count` (and `printCount`) instead of `index` since it starts from 1. Like "disable circular dependencies counting" describes this flag better.

Reply:
Like this? Sorry, but I just don't get the "commander" tool. How this is parsed and how the variables are extracted and set. This is a trial and error for me